js/app.js

```diff
@@ -582,6 +582,14 @@
 			this.connection = new OCA.Talk.Connection(this);
 			this.token = $('#app').attr('data-token');
 
+			this.signaling.on("joinRoom", function(/* token */) {
```

Comment:
single quotes? :see_no_evil:

Reply:
Whoops :see_no_evil:
You should probably enforce it by eslint if you always want to have single quotes in JS :speak_no_evil: 

Anyway, changed and squashed.